index.html

```diff
@@ -66,14 +66,191 @@ <h1>I’m Tracy<br />Glad to see,You!</h1>
     </section>
     <section class="grid-container">
       <div class="item1">
-        <div class="item1a"></div>
+        <div class="item1a">
+          <img src="./images/snapshoot.png" />
+        </div>
+        <div class="item1b">
+          <h1>Tonic</h1>
+          <div class="frame2">
+            <span>CANOPY</span>
+            <ul class="role">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="tags">
+            <ul>
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+            <button><a href="">See Project</a></button>
+          </div>
+        </div>
+      </div>
+      <div class="item2">
+        <div class="item2a">
+          <img src="./images/snap2.png" />
+        </div>
+        <div class="item1b">
+          <h1>Multi-Post Stories</h1>
+          <div class="frame2">
+            <span>CANOPY</span>
+            <ul class="role">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="tags">
+            <ul>
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+            <button><a href="">See Project</a></button>
+          </div>
+        </div>
+      </div>
+      <div class="item3">
+        <div class="item3a">
+          <img src="./images/snap3.png" alt="tracy" />
+        </div>
         <div class="item1b">
           <h1>Tonic</h1>
+          <div class="frame2">
+            <span>CANOPY</span>
+            <ul class="role">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="tags">
+            <ul>
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+            <button><a href="">See Project</a></button>
+          </div>
+        </div>
+      </div>
+      <div class="item4">
+        <div class="item4a">
+          <img src="./images/snap4.png" alt="tracy" />
         </div>
+        <div class="item1b">
+          <h1>Multi-Post Stories</h1>
+          <div class="frame2">
+            <span>CANOPY</span>
+            <ul class="role">
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
```

Comment:
- [ ] Kindly reduce the space between `frame2` class and `role` class so it matches the Figma design

Your design
![Screenshot from 2023-01-13 14-53-11](https://user-images.githubusercontent.com/80458199/212326530-c09fefe7-16cc-45b9-9dac-68c46c44234a.png)


Figma design

![Screenshot from 2023-01-13 15-01-28](https://user-images.githubusercontent.com/80458199/212326529-16c1106a-70b7-4e5d-9b4c-1b636354252d.png)